pythran/config.py

```diff
@@ -149,7 +157,17 @@ def parse_define(define):
 
     for k, w in extra.items():
         extension[k].extend(w)
-    if cfg.getboolean('pythran', 'complex_hook'):
+
```

Comment:
Instead of adding the complexity here, I think you should udpate ``init_cfg`` to accept the extra config argument. And as you asked previously: they are more important than any config file.

Reply:
OK I"ll try that.

Reply:
mmm maybe I'm missing something but init_cfg(sys_file, platform_file, user_file) is called at module load time 
```
cfg = init_cfg('pythran.cfg',
               'pythran-{}.cfg'.format(sys.platform),
               '.pythranrc')
```
A that point, I don't see how I can pass the extra config arguments.

Reply:
OK Take a look at it now. I think you're right, it's a lot cleaner.